src/lib/db/access-store.ts

```diff
@@ -204,7 +204,7 @@ export class AccessStore implements IAccessStore {
         let userPermissionQuery = this.db
             .select(
                 'project',
-                'permission',
+                'p.permission',
```

Comment:
Should we proactively do the same for other columns?

Reply:
I agree that we can fix this ASAP and then try to address this later as a PM action.